packages/calcite-components/src/components/list-item/list-item.scss

```diff
@@ -66,6 +66,10 @@
   &::before {
     background-color: var(--calcite-list-selection-border-color, var(--calcite-color-brand));
   }
+
+  &:focus {
+    box-shadow: inset 4px 0 0 0 var(--calcite-list-selection-border-color, var(--calcite-color-brand));
```

Comment:
Should the offset value (not sure about blur and spread) use a token? cc @ashetland @macandcheese

Reply:
It could use `--calcite-border-width-lg`

Reply:
In this case it could make sense. The point of using the box-shadow was to mimic a border, but not take up space.